applications/nrf5340_audio/boards/nrf5340dk_nrf5340_cpuapp.overlay

```diff
@@ -16,3 +36,37 @@
 		hp-channel-r;
 	};
 };
+
+&pinctrl {
+	i2s0_default: i2s0_default {
+		group1 {
+			psels = <NRF_PSEL(I2S_MCK, 0, 12)>;
+			nordic,drive-mode = <NRF_DRIVE_H0H1>;
+		};
+		group2 {
+			psels = <NRF_PSEL(I2S_SCK_M, 0, 14)>,
+			        <NRF_PSEL(I2S_LRCK_M, 0, 16)>,
+			        <NRF_PSEL(I2S_SDOUT, 0, 13)>,
+			        <NRF_PSEL(I2S_SDIN, 0, 15)>;
+		};
+	};
+
+	i2s0_sleep: i2s0_sleep {
```

Comment:
Why are there two groups of I2S pins?

Reply:
Well I copied this from the board file for the audio DK, but I assume it is because the master clock does not have to be driven (in or out).
The remaining pins in the second group are required as a minimum for I2S.

Reply:
Discuss

Reply:
I believe this is to cover the two modes of I2S, master or slave?